lib/pbench/server/api/resources/datasets_compare.py

```diff
@@ -74,46 +72,37 @@ def _get(
         """
 
         datasets = params.query.get("datasets")
-        authorized_user: User = Auth.token_auth.current_user()
-
-        if not authorized_user:
-            raise APIAbort(
-                HTTPStatus.UNAUTHORIZED,
-                "User provided access_token is invalid or expired",
-            )
-
-        cache_m = CacheManager(self.config, current_app.logger)
-        benchmark = None
-        benchmark_check = set()
-        stream_file = {}
-
+        benchmark_choice = None
         for dataset in datasets:
-            benchmark = Metadata.getvalue(dataset, "dataset.metalog.pbench.script")
-            benchmark_check.add(benchmark)
-
+            benchmark = Metadata.getvalue(
+                dataset=dataset, key="dataset.metalog.pbench.script"
+            )
```

Comment:
I liked it better without the keywords on one line, but this works too. 😆

Reply:
Yeah...without the keywords, Black didn't have to wrap the line for us....  🥴